### ISSUE TITLE
Get availability script not working

### ISSUE THREAD

**Issue by arnav0103** [gsidhu/cpcbccr-data-scraper#2]
Hi,

I am trying to use the script get_availability to fetch the year assessment and ended up with error Unauthorized access.

Can you please help. Have u tried the script recently. Any insights would help a lot.

**Comment by gsidhu** [comment]
Hi Arnav,

The CPCB seems to have updated their dashboard. I am not sure if it has had an impact on their backend API but it could have. 

Perhaps they have gotten rid of the `Year_summarization` endpoint – which is what the get_availability script targets.

Have you tried using the other scripts to fetch data directly?

If you get unauthorised access there as well, they have likely added a layer of security to the API requests. Check if the headers in the network request now have a cookie (which would probably authorise the session).

I am not able to commit time to this project at the moment but I would appreciate you make a PR if you are able to fix it!

Cheers,
G

**Comment by NoOne-play** [comment]
yes the header have cookie. but how to implement it in a code. i am not a programmer so i have no idea, can you please guide me how to do this. and cookies also contain captcha

**Comment by NoOne-play** [comment]
> yes the header have cookie. but how to implement it in a code. i am not a programmer so i have no idea, can you please guide me how to do this. and cookies also contain captcha

I try putting cookies in the header section, I also tried using selenium to get cookies and also solve the captcha manually, but still gets unauthorised request. Any help please.
